.github/workflows/lint.yaml

```diff
@@ -0,0 +1,20 @@
+name: Lint
+
+on:
+  pull_request:
+
+jobs:
+  actionlint:
+    name: actionlint
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        shell: bash
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Check workflow files
+        run: |
+            echo "::add-matcher::.github/actionlint-matcher.json"
+            bash <(curl https://raw.githubusercontent.com/rhysd/actionlint/main/scripts/download-actionlint.bash)
```

Comment:
Please pin it to the newest version (`v1.6.10`) instead of using main

Reply:
looks like the latest version is not compatible with our workflow `outputs` (https://github.com/bakdata/ci-templates/runs/5625138460)
it's fixed in main https://github.com/rhysd/actionlint/commit/77d8de26cbe771ed04028ff9da3b147deb39e187

Reply:
I've pinned it to this commit. When the next version is released we can pin the tag again.